ctapipe/reco/sklearn.py

```diff
@@ -83,6 +83,9 @@ class SKLearnReconstructor(Reconstructor):
     #: Name of the target column in training table
     target: str = None
 
+    prefix = Unicode(
+        default_value=None, allow_none=True, help="Prefix for output of this model"
```

Comment:
Add that if None, the configured model_cls name will be used

Reply:
Is it possible to make default_value=model_cls?